### ISSUE TITLE
Add more discord actions

### ISSUE THREAD

**Issue by SrPandaStick** [CraftingStore/Suggestions#364]
When someone buys a temporary package and receives a discord role at the time of expiration if the package still has the role and we have to modify this manually.

Add more actions for discord, like normal for servers.

**Comment by EnderGamingFilms** [comment]
I would like to see this but after using the discord option I realize that it does not modify the user's account immediately they have to run a command in the discord so the crafting store bot knows which user to modify. The only way this idea could work is if the user entered their discord account information at checkout.

**Comment by Tim-NL** [comment]
Hi,

Thanks for the suggestions!

We've now implemented two new features;
1. The user can now login with discord on checkout (see: https://help.craftingstore.net/books/general/page/discord-integration)
2. When the user still uses the old system, and does not login on checkout, they'll get a mail like the would before. And when they do !claim. We store their Discord ID so we can run expiry actions.

So in short; Both of these have been implemented. 

-Tim
